packages/relay/tests/lib/eth/eth_estimateGas.spec.ts

```diff
@@ -165,6 +165,11 @@ describe('@ethEstimateGas Estimate Gas spec', async function () {
   });
 
   it('should eth_estimateGas transfer to existing account', async function () {
+    const callData = {
+      to: RECEIVER_ADDRESS,
+      value: 10, //in tinybars
+    };
+    mockContractCall(callData, true, 501, { errorMessage: '', statusCode: 501 });
```

Comment:
Same here and for the other tests below, why do we need to do this mock contract call with 501? Isn't this a positive scenario?

Reply:
The problem here is that tests have not been refactored for the new changes in estimate gas. We used to be able to get estimation from mirror-node only for read-only smart contract calls and if the call was a transfer for example, we would return the baseCost. Thus, this test is written in this way. However, now we can also estimate transfers and the test and is no more relevant and needs to be refactored. 
Good catch : )